### DISCUSSION TITLE
Hi everyone

### DISCUSSION THREAD

**Discussion by random9** [community/community#36349] [New to GitHub]
I'm newbie around here! 

I know some basics github
And I have some bases in python, that I want to put in practice, and sense github have a big community and is a open source, I want to try to be more ative in fixe issues in another projects, if you guys have some advice for me please, and where should I start please go ahead, I Apriciate

**Comment by melvincwng** [comment]
Hi @random9, nice meeting you on GitHub 😊

If you wish to contribute to a beginner friendly open source project, you can consider taking a look at [Your First Open Source Project](https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project).

Hope this helps 🎉

**Comment by ossanna16** [comment]
Hey there @random9 and thank you for asking a good question! I'm glad you found our community 🙂
